blocks/single-column-layout-block/_index.scss

```diff
@@ -9,6 +9,7 @@
 	width: 100%;
 	margin: auto;
 	@include scss.block-properties("single-column-regular");
+	@include scss.block-components("single-column-regular");
 }
 
 .b-single-column-regular-footer {
```

Comment:
Should we BEM these classes?

Reply:
will work on this 👀

Reply:
cool updated -- thanks https://github.com/WPMedia/arc-themes-blocks/pull/1334/commits/68e8ea61f0bbfe63e3cd0edfa06f2265bf0a51b4